web/html/src/branding/css/base/theme.scss

```diff
@@ -1332,6 +1332,8 @@ tr.changed > td {
 
 .badge {
   font-size: 12px;
+  border-radius: 10px;
+  background: $eos-bc-gray-1000;
```

Comment:
I'm guessing this should probably be `$badge-bg`?

Reply:
Fixed